pkg/network/ebpf/kprobe_types.go

```diff
@@ -53,3 +54,10 @@ const (
 )
 
 const BatchSize = C.CONN_CLOSED_BATCH_SIZE
+
+var (
+	StaticTags = map[uint64]string{
+		C.LIBGNUTLS: "tls.library:gnutls",
+		C.LIBSSL:    "tls.library:openssl",
```

Comment:
I think we should have a type for the tag values, so we can use them in userspace too.
```
type ConnTag uint64
const (
    GnuTLS ConnTag = C.LIBGNUTLS
    OpenSSL ConnTag = C.LIBSSL
)
var (
    StaticTags = map[ConnTag]string{
        GnuTLS: "tls.library:gnutls",
        OpenSSL: "tls.library:openssl",
    }
```